Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift

```diff
@@ -65,15 +66,20 @@ class ConfirmYourBidPINViewController: UIViewController {
                         .updateUserCredentials(loggedInProvider)
                         .mapReplace(provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { provider -> Observable<BypassResults> in
                     return me
-                        .fulfillmentNav()
-                        .updateUserCredentials(loggedInProvider)
-                        .mapReplace(provider)
+                        .viewModel
+                        .checkForAdminCCBypass(bidDetails.auctionID, authorizedNetworking: provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { results -> Observable<Void> in
+                    // Check if we should bypass the CC requirement and move directly onto the next step.
+                    guard results.bypassCCRequirement == false else {
```

Comment:
This feels weird because I should've been using an enum with explicit cases instead of true/false. I think I was too distracted trying to get an improvement to the `.mapReplace(provider)` pattern :wink: Definitely merits more thought.